paddle/scripts/submit_local.sh.in

```diff
@@ -91,34 +88,30 @@ else:
   sys.exit(0)
 EOF
 
-if [ $? -eq 1 ]; then  # Older version installed, or not installed at all
-    echo "First time run paddle, need to install some python dependencies."
-    # setuptools normalizes package version, so we need to use normalized
-    # package version for paddle python package
-    PYTHON_PADDLE_VERSION=$(python -c 'import packaging.version
-import setuptools
-print str(packaging.version.Version("@PADDLE_VERSION@"))
-' 2>/dev/null)
-    BASEDIR=$(dirname "$0")
-    pip install ${BASEDIR}/../opt/paddle/share/wheels/*-${PYTHON_PADDLE_VERSION}-*.whl
-    if [ $? -ne 0 ]; then
-	echo "pip install wheels failed. "
-	echo "Please use 'sudo paddle' at the first time you use PaddlePaddle"
-	echo "PaddlePaddle will install some python dependencies automatically."
-	exit 1
+# find the paddle executable module path
+PYTHON_SITEPACKAGES_PATHES=$(python -c "import site; print ';'.join(site.getsitepackages())")
+IFS=';' items=($PYTHON_SITEPACKAGES_PATHES)
+for item in ${items[@]}; do
+    if [ -f $item/usr/local/opt/paddle/bin/paddle_trainer ]; then
+        PADDLE_BIN_PATH=$item/usr/local/opt/paddle/bin
```

Comment:
Yep, it's also confusing for me, `print sys.prefix` will print `/usr`. 
```bash
root@6a4bc12ae305:/# python -c "import sys; print sys.prefix"
/usr
```

I executed `pip install *.whl` in [production Docker image](https://github.com/PaddlePaddle/Paddle/tree/develop/paddle/scripts/docker#build-the-production-docker-image), the base image is `ubuntu:16.04`.

And if the data_files was configured as `local/opt/paddle/bin`, the binary files would be installed under `/usr/local/local/opt/paddle/bin`, the duplicate `local` directory.